introduction.md

```diff
@@ -1,2 +1,13 @@
 Hi, my name is Darko Minarik
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?